src/ast/values.rs

```diff
@@ -107,7 +108,8 @@ impl<'a> fmt::Display for Value<'a> {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
```

Comment:
It's weird to me we have a Display impl for Value. I don't remember where we use it, is there a specific purpose to it?

Reply:
the logger...